client/lib/wp/localization/index.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * External dependencies
+ */
+import qs from 'querystring';
+
+/**
+ * Internal dependencies
+ */
+import { getCurrentUserLocale } from 'state/current-user/selectors';
+
+/**
+ * Module variables
+ */
+let locale;
+
+/**
+ * Given a WPCOM parameter set, modifies the query such that a non-default
+ * locale is added to the query parameter.
+ *
+ * @param  {Object} params Original parameters
```

Comment:
The hyphen itself is optional, and is available to help readability ([reference](http://usejsdoc.org/tags-param.html)), though a similar effect is achieved via aligning spaces on the start of the type, name, and description.

Usage seems a bit mixed throughout Calypso. In my own case, I'm using the [Sublime Text 3 DocBlockr package](https://github.com/spadgos/sublime-jsdocs), which does not add the hyphens and (as far as I can tell) does not offer an option to do so, so it's a bit more effort for me to revise. _TL;DR:_ I'm lazy :laughing: